src/frontend/js/widgets/Dashboard/components/DashboardLayoutRoute/index.tsx

```diff
@@ -8,11 +9,15 @@ export const DashboardLayoutRoute = () => {
     const handle = match.handle || {};
     return !!(handle as DashboardRouteHandle).renderLayout;
   });
-  return renderOutletOnly ? (
-    <Outlet />
-  ) : (
-    <DashboardLayout>
-      <Outlet />
-    </DashboardLayout>
+  return (
+    <CunninghamProvider>
```

Comment:
I got an error on `yarn build-ts` that disappears when i comment `CunninghamProvider` : 


```shell
ERROR in ./js/widgets/Dashboard/components/DashboardLayoutRoute/index.tsx 4:0-63
Module not found: Error: Default condition should be last one
 @ ./js/widgets/Dashboard/utils/dashboardRoutes.tsx 37:0-89 48:31-51
 @ ./js/widgets/Dashboard/hooks/useDashboardRouter/index.tsx 4:0-100 14:29-47 23:21-42
 @ ./js/widgets/Dashboard/index.tsx 7:0-60 13:68-86
 @ ./js/widgets/index.tsx 68:9-36
 @ ./js/index.tsx 49:0-31 182:42-46
```

Reply:
[Here](https://github.com/openfun/cunningham/pull/50) a PR on cunningham that should fix it.